ResearchUI/ResearchUI/iOS/RSDStepViewController.swift

```diff
@@ -415,6 +414,8 @@ open class RSDStepViewController : UIViewController, RSDStepViewControllerProtoc
                     fetchLoader.fetchImage(for: imageView.bounds.size, callback: { [weak navigationView] (_, img) in
                         navigationView?.image = img
                     })
+                } else {
+                    navigationView.imageView?.image = UIImage(named: imageTheme.imageIdentifier)
```

Comment:
Images from image themes were only being loaded if they were animated or fetched from the web